src/ddprof_worker.cc

```diff
@@ -236,9 +237,12 @@ DDRes aggregate_livealloc_stack(
     const LiveAllocation::PprofStacks::value_type &alloc_info,
     DDProfContext &ctx, const PerfWatcher *watcher, DDProfPProf *pprof,
     const SymbolHdr &symbol_hdr) {
-  DDRES_CHECK_FWD(
-      pprof_aggregate(&alloc_info.first, symbol_hdr, alloc_info.second._value,
-                      alloc_info.second._count, watcher, kLiveSumPos, pprof));
+  DDProfValuePack pack{
+      alloc_info.second._value,
+      static_cast<uint64_t>(std::max<int64_t>(0, alloc_info.second._count)), 0};
```

Comment:
As discussed, the time here could be tied to the time of allocation.
Though this would not have immediate value for the front end (only diagnostic value for the cases where we open the pprofs).

Reply:
We will postpone the timeline allocation story for now. Other runtimes have not specified this.